TTBot/Models/ChampionshipPointsModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TTBot.Models
+{
+    class ChampionshipPointsModel
```

Comment:
Future enhancement here to be able to add custom points format for a championship maybe?

Reply:
Yes, let's push that in to iteration 2.